### PR TITLE
Add descriptions for remaining event handling examples

### PR DESCRIPTION
**Pull request by divyam3897**
Add one line descriptions for event handling examples
Part of https://github.com/matplotlib/matplotlib/issues/8921
## PR Checklist

- [ ] Has Pytest style unit tests
- [ x] Code is PEP 8 compliant 
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: examples/event_handling/close_event.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 Close Event
 ===========
 
+Show how to connect to close events
```
Comment (choldgraf) [inline examples/event_handling/close_event.py]: add a period!

also, could you make it a bit more clear? E.g., "Connecting events that occur when the figure closes."

File: examples/event_handling/data_browser.py [hunk 2]
```diff
@@ -3,6 +3,8 @@
 Data Browser
 ============
 
+An example of how to interact with multiple canvas by selecting and highlighting
```
Comment (choldgraf) [inline examples/event_handling/data_browser.py]: I'd split this into two blocks. The first sentence should be quite short and fit one one line. Something like:

```
Connecting data between multiple canvases.

This example covers how to interact data with multiple canvases. This
let's you select and highlight a point on one axis, and generating the
data of that point on the other axis.
```

File: examples/event_handling/looking_glass.py [hunk 3]
```diff
@@ -3,6 +3,7 @@
 Looking Glass
 =============
 
+Example showing the movement of a glass to view the points using mouse events on a canvas.
```
Comment (choldgraf) [inline examples/event_handling/looking_glass.py]: "Using mouse events to simulate a looking glass for inspecting data."

File: examples/event_handling/path_editor.py [hunk 4]
```diff
@@ -3,6 +3,9 @@
 Path Editor
 ===========
 
+This is an example showing a cross-GUI application using
```
Comment (choldgraf) [inline examples/event_handling/path_editor.py]: something like...
```
Sharing events across GUIs.

This example demonstrates a cross-GUI application using Matplotlib event
handling to interact with and modify objects on the canvas.
```

**Comment by choldgraf** [comment]
these are a great start! some suggestions for wording etc in the comments, then I think we're good to go

**Comment by divyam3897** [comment]
@choldgraf made the changes 👍

**Comment by choldgraf** [comment]
LGTM - will merge when tests finish

**Comment by tacaswell** [comment]
The failures are due to circle testing the tip of the branch, not it merged into master.  The version of master this is based off of is pre #9325 

Who ever merges this should confirm the docs build locally (probably less work over all unless @divyam3897 is comfortable rebasing the branch onto current master).

**Comment by divyam3897** [comment]
@tacaswell I just pushed the changes after the rebase, I hope the tests should pass now :)

**Comment by tacaswell** [comment]
@divyam3897 Thanks!

**Comment by tacaswell** [comment]
Thanks @divyam3897 and congrats on your first Matplotlib contribution :tada: 

Hopefully we will hear from you again!

**Comment by tacaswell** [comment]
backported to v2.1.0-doc as 1e6dc92ce03ed3d918e5b0eb211725039370969a

**Pull request merged**